pyproject.toml

```diff
@@ -85,6 +85,10 @@ sqlalchemy = {version = "^1.1.0", optional = true}
 
 statsforecast = {version = "1.4", optional = true}
 
+transformers = {version = "<5", optional = true}
+accelerate = {version = "<1", optional = true}
+huggingface-hub = {version = ">=0.21,<0.23", optional = true}
```

Comment:
I don't see it being used below in defining chronos extra.

Reply:
`huggingface-hub` is already dependency of `accelerate `, so here I only limit version range. Do we need to define extra with it?

Reply:
Why do we need to limit the version of it?

Reply:
I had some strange bugs (like [here](https://stackoverflow.com/questions/71481711/oserror-for-huggingface-model)) connected with loading model on other versions.

Reply:
- As I understand it is a dependency of `transformers` package, not `accelerate`. 
- Do we really need accelerate? Where are we using it?

Reply:
- Ok, it is dependency of both.
- As I remember I tried to install only `transformers` first, but it requested `accelerate` in some inner imports
![image](https://github.com/user-attachments/assets/0531f2da-8431-45e7-a2f7-943f63321141)

Reply:
- Let's add it into chronos extras
- Let's add it into `settings.py`

It is for consistency.

Reply:
Added